yt_dlp/utils.py

```diff
@@ -3439,7 +3441,7 @@ class OVERLAPPED(ctypes.Structure):
     whole_low = 0xffffffff
     whole_high = 0x7fffffff
 
-    def _lock_file(f, exclusive):
+    def _lock_file(f, exclusive, block=None):  # todo: block unused on win32
```

Comment:
keep the signatures exactly same. So no `=None`

Reply:
ah, yes, fixed.